src/App.module.css

```diff
@@ -1,12 +1,41 @@
+:root {  
+  --color-dark-main: #354B61;
+  --color-ligth-main: #FFFFFF;
+  --color-ligth-main-second:#F6F6F6;
+  --color-border-table: #2C3E50;
+  --color-header: #008080;
+}
+
+button {
+  cursor: pointer;
+}
+button:disabled {
+  cursor: not-allowed;
+}
+
+html,
+body {
+  height: 98vh;
+  width: 100vw;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
 .App {
+  height: 100%;
+  width: 100%;
   display: flex;
-  min-height: 100vh;
+  /* min-height: 100vh; */
```

Comment:
Would it also work the same if the `height: 98vh` is applied here instead of in the "root" div in index.html?